index.html

```diff
@@ -43,104 +53,131 @@ <h1>Hey there. I'm <br> Hiwot<span class="h1color"><br>
                     <h4>Multi-Post Stories Gain+Glory</h4>
-                    <ul class="button-detail"> 
+                    <ul class="button-detail" id="desktop-button-detail"> 
                         <li>Ruby on rails</li>
-                         <li>css</li>
+                        <li>css</li>
                         <li>JavScript</li>
                         <li>html</li>
                     </ul>
-                     <button class="see-project">See project </button>
-                 </div> 
+                    <button class="see-project" id="desktop-see-project">See project </button>
+                </div>  
             </div>
-            <div class="project4">  
-                 <div class="project-detail">
+            <div class="project4" id="desktop-project4">  
+                <div class="project-detail" id="desktop-project-detail">
                     <h4>Multi-Post Stories Gain+Glory</h4>
-                    <ul class="button-detail"> 
+                    <ul class="button-detail" id="desktop-button-detail"> 
                         <li>Ruby on rails</li>
                         <li>css</li>
                         <li>JavScript</li>
                         <li>html</li>
                     </ul>
-                    <button class="see-project">See project </button>
+                    <button class="see-project" id="desktop-see-project">See project </button>
                 </div> 
             </div>
-             <div class="project5">  
-                <div class="project-detail">
+             <div class="project5" id="desktop-project5">  
+                <div class="project-detail" id="desktop-project-detail">
                     <h4>Multi-Post Stories Gain+Glory</h4>
-                    <ul class="button-detail"> 
+                    <ul class="button-detail" id="desktop-button-detail"> 
                         <li>Ruby on rails</li>
                         <li>css</li>
                         <li>JavScript</li>
                         <li>html</li>
                     </ul>
-                    <button class="see-project"> <a href="https://github.com/hiwina"></a> See project </button>
+                    <button class="see-project" id="desktop-see-project">See project </button>
                 </div> 
             </div>
-            <div class="project6">  
-                <div class="project-detail">
+            <div class="project6" id="desktop-project6">   
+                <div class="project-detail" id="desktop-project-detail">
                     <h4>Multi-Post Stories Gain+Glory</h4>
-                    <ul class="button-detail"> 
+                    <ul class="button-detail" id="desktop-button-detail"> 
                         <li>Ruby on rails</li>
                         <li>css</li>
                         <li>JavScript</li>
                         <li>html</li>
                     </ul>
-                    <button class="see-project">See project </button>
```

Comment:
- [ ] Kindly make sure that you followed the button functionality ( hover, press ....). its good for your UX experience and design.
![hoverdesign](https://user-images.githubusercontent.com/66992901/199488235-e58f629b-6dbb-420c-a1a6-224113d6a2d1.PNG)